src/python/espressomd/_system.pyx

```diff
@@ -245,7 +245,7 @@ cdef class System:
         for i in range(n_nodes):
             states_on_node_i = []
             for j in range(_state_size_plus_one + 1):
-                states_on_node_i.append(rng.randint(0, sys.maxint))
+                states_on_node_i.append(rng.randint(0, 2**31-1))
```

Comment:
Better extend `numeric_limits[T]` in `utils.pxd` by `max()`, i.e.
````python
cdef extern from "<limits>" namespace "std" nogil:
    cdef cppclass numeric_limits[T]:
        @staticmethod
        T epsilon()
        @staticmethod
        T max()
````
and then use
````
numeric_limits[int].max
````
This will translate to C++ code `std::numeric_limits<int>::max()` which will adjust to the `int` chosen by the compiler.